src/Illuminate/Database/Eloquent/Relations/Relation.php

```diff
@@ -154,6 +154,10 @@ abstract public function getResults();
      */
     public function getEager()
     {
+        if ($this->eagerImplicitlyEmpty ?? false) {
```

Comment:
Where is `eagerImplicitlyEager` defined?

Reply:
Have now added that as a proper parameter along with docblock comment.